hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/SourceFormatAdapter.java

```diff
@@ -49,9 +57,119 @@
 public final class SourceFormatAdapter implements Closeable {
 
   private final Source source;
+  private final SourceFormatAdapterConfig config;
+
+  public static class SourceFormatAdapterConfig extends HoodieConfig {
```

Comment:
lets add config annotation.
lets move this to a separate class and not embed here. 
also, may be we should name this DeltaStreamerSourceConfigs.